### PR TITLE
Gk/pnapi js

### PR DESCRIPTION
**Pull request by garvank**
@mikeweaver these are two new files covering SolutionsJS and the "Toolkit" library provided by SolutionsJS. Could you please CR? Thank you!

### CODE REVIEW

File: source/web_integration/index.rst [hunk 1]
```diff
@@ -6,8 +6,10 @@ Web Integration dynamically displays a unique phone number for each source drivi
 See the following technical documentation for details on how to setup Web Integration:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 4
```
Comment (mikeweaver) [inline source/web_integration/index.rst]: I don't think you want to increase the depth. This is not the number of files listed below, but the number of levels that will be in the table of contents.

Comment (garvank) [inline source/web_integration/index.rst]: Reverted!

**Comment by mikeweaver** [comment]
Couple of things:
1) It looks like you might have branched off master, not 2016-05-01? I merged 2016-05-01 into your branch for you.
2) There are errors in your build:
/Users/mweaver/invoca/developer-docs/source/web_integration/pnapi_web_integration.rst:53: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/Users/mweaver/invoca/developer-docs/source/web_integration/pnapi_web_integration.rst:56: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/Users/mweaver/invoca/developer-docs/source/web_integration/pnapi_web_integration.rst:59: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/Users/mweaver/invoca/developer-docs/source/web_integration/pnapi_web_integration.rst:341: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/Users/mweaver/invoca/developer-docs/source/web_integration/pnapi_web_integration.rst:347: WARNING: Enumerated list ends without a blank line; unexpected unindent.

**Comment by garvank** [comment]
@spencerpbartz @mikeweaver fixed!

**Comment by mikeweaver** [comment]
:+1:

**Pull request merged**